include/pidfd-utils.h

```diff
@@ -31,6 +31,13 @@ static inline int pidfd_open(pid_t pid, unsigned int flags)
 }
 #  endif
 
+#  ifndef HAVE_PIDFD_GETFD
+static inline int pidfd_getfd(int pidfd, int targetfd, unsigned int flags)
+{
+	return syscall(SYS_pidfd_open, pidfd, targetfd flags);
```

Comment:
```
return syscall(SYS_pidfd_getfd, pidfd, targetfd, flags);
```
?

Reply:
https://github.com/util-linux/util-linux/pull/3202/commits/7107a5c6a823460ad576660a69e994bc1f4aeecb

Reply:
Wow, I'm surprised how I missed this copy'n'paste. I'm pretty sure I test every time before pushing, but somehow that slipped in. Thanks!